crates/bevy_dev_tools/src/hotpatch.rs

```diff
@@ -0,0 +1,42 @@
+//! Utilities for hotpatching code.
+
+use bevy_ecs::{event::EventWriter, HotPatched};
+use dioxus_devtools::{connect, subsecond::apply_patch, DevserverMsg};
+
+pub use dioxus_devtools::subsecond::{call, HotFunction};
+
+use crate::{Last, Plugin};
+
+/// Plugin connecting to Dioxus CLI to enable hot patching.
+#[derive(Default)]
+pub struct HotPatchPlugin;
+
+impl Plugin for HotPatchPlugin {
+    fn build(&self, app: &mut crate::App) {
+        let (sender, receiver) = crossbeam_channel::bounded::<HotPatched>(1);
+
+        // Connects to the dioxus CLI that will handle rebuilds
+        // On a successful rebuild the CLI sends a `HotReload` message with the new jump table
+        // When receiving that message, update the table and send a `HotPatched` message through the channel
+        connect(move |msg| {
```

Comment:
`subsecond` seems to provide hot-reload handler, could it be used here?

```rust
use dioxus_devtools::{connect_subsecond, subsecond::register_handler};

...
        connect_subsecond();
        register_handler(Arc::new(move || {
            sender.send(HotPatched).unwrap();
        }));
```

Reply:
done, thanks! 👍